code/modules/reagents/chemistry/reagents/drinks/drink_reagents.dm

```diff
@@ -0,0 +1,912 @@
+	description = "Blows right through you like a space wind."
+	color = "#102000" // rgb: 16, 32, 0
+	taste_description = "sweet citrus soda"
+	adj_drowsy = -7
+	adj_sleepy = -1
+	adj_temp = - 5
+	targ_temp = BODYTEMP_NORMAL - 5
+
+/datum/reagent/consumable/dr_gibb
+	name = "Dr. Gibb"
+	description = "A delicious blend of 42 different flavours."
+	color = "#102000" // rgb: 16, 32, 0
+	taste_description = "cherry soda" // FALSE ADVERTISING
+	adj_drowsy = -6
+	adj_temp = - 5
+	targ_temp = BODYTEMP_NORMAL - 5
+
+/datum/reagent/consumable/space_up
+	name = "Space-Up"
+	description = "Tastes like a hull breach in your mouth."
+	color = COLOR_VIBRANT_LIME // rgb: 0, 255, 0
+	taste_description = "cherry soda"
+	adj_temp = - 8
+	targ_temp = BODYTEMP_NORMAL - 10
+
+/datum/reagent/consumable/lemon_lime
+	name = "Lemon Lime"
+	description = "A tangy substance made of 0.5% natural citrus!"
+	color = "#8CFF00" // rgb: 135, 255, 0
+	taste_description = "tangy lime and lemon soda"
+	adj_temp = - 8
+	targ_temp = BODYTEMP_NORMAL - 10
+
+/datum/reagent/consumable/pwr_game
+	name = "Pwr Game"
+	description = "The only drink with the PWR that true gamers crave."
+	color = "#9385bf" // rgb: 58, 52, 75
+	taste_description = "sweet and salty tang"
+
+/datum/reagent/consumable/shamblers
+	name = "Shambler's Juice"
+	description = "~Shake me up some of that Shambler's Juice!~"
+	color = "#f00060" // rgb: 94, 0, 38
+	taste_description = "carbonated metallic soda"
+
+/datum/reagent/consumable/sodawater
+	name = "Soda Water"
+	description = "A can of club soda. Why not make a scotch and soda?"
+	color = "#619494" // rgb: 97, 148, 148
+	taste_description = "carbonated water"
+	adj_dizzy = -5
+	adj_drowsy = -1
+	adj_temp = - 5
+	targ_temp = BODYTEMP_NORMAL - 5
+
+/datum/reagent/consumable/tonic
+	name = "Tonic Water"
+	description = "It tastes strange but at least the quinine keeps the Space Malaria at bay."
+	color = "#0064C8" // rgb: 0, 100, 200
+	taste_description = "tart and fresh"
+	adj_dizzy = - 5
+	adj_drowsy = -2
+	adj_sleepy = -1
+	adj_temp = - 5
+	targ_temp = BODYTEMP_NORMAL - 5
+
+/datum/reagent/consumable/wellcheers
+	name = "Wellcheers"
+	description = "A strange purple drink, smelling of saltwater. Somewhere in the distance, you hear seagulls."
+	color = "#762399" // rgb: 118, 35, 153
+	taste_description = "grapes and the fresh open sea"
+
+/datum/reagent/consumable/monkey_energy
+	name = "Monkey Energy"
+	description = "The only drink that will make you unleash the ape."
+	color = "#f39b03" // rgb: 243, 155, 3
+	overdose_threshold = 60
+	taste_description = "barbecue and nostalgia"
+
+/datum/reagent/consumable/ice
+	name = "Ice"
+	description = "Frozen water, your dentist wouldn't like you chewing this."
+	reagent_state = SOLID
+	color = "#619494" // rgb: 97, 148, 148
+	taste_description = "ice"
+	default_container = /obj/item/reagent_containers/cup/glass/ice
+	adj_temp = - 7
+	targ_temp = BODYTEMP_NORMAL - 15
+
+/datum/reagent/consumable/soy_latte
```

Comment:
Our versions of Coffee Latte and Soy Latte are duplicated on line 818 and 830, theirs doesn't have the Healing effects that ours does, as well as ours are childed/branced under Coffee. (muh power game)

Reply:
respond to above comment pls